pkg/api/customization/alert/notifier_action.go

```diff
@@ -203,8 +203,8 @@ func testWebhook(url, msg string) error {
 	}
 	defer resp.Body.Close()
 
-	if resp.StatusCode != http.StatusOK {
-		return fmt.Errorf("http status code is not 200")
+	if resp.StatusCode < http.StatusOK || resp.StatusCode >= http.StatusMultipleChoices {
+		return fmt.Errorf("http status code is %d, not 200", resp.StatusCode)
```

Comment:
@aiwantaozi the error message is kind of misleading - is the correct response should always be 200? Then why not check it in the if condition this way? like response != 200. If the response is meant to be within some range, the error message should indicate that.